xml/System.Drawing/SystemColors.xml

```diff
@@ -1344,9 +1344,12 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets whether the alternative color set is used.</summary>
+        <value>When <see langword="true"/>, system <see cref="KnownColor"/> values will return the alternative color set (as returned by <see cref="SystemColors"/> statics or <see cref="Color.FromKnownColor(KnownColor)"/>). This is currently "dark mode" variants of the system colors.</value>
```

Comment:
These `see crefs` are going to fail the CI. They need to be in DocID format (full name).

Reply:
```suggestion
        <value>When <see langword="true"/>, system <see cref="T:System.Drawing.KnownColor"/> values will return the alternative color set (as returned by <see cref="T:System.Drawing.SystemColors"/> statics or <see cref="M:System.Drawing.Color.FromKnownColor(System.Drawing.KnownColor)"/>). This is currently "dark mode" variants of the system colors.</value>
```

Reply:
Thanks for the catch!

Reply:
No problem. Did you use the porting tool to bring in these comments? If yes, then there's a bug in the tool and I have to fix it.

Reply:
No, this was my mistake. I thought it might be quicker for me to port these over manually since there were only a handful left -- should've used the tool 😄